app/pages/project/stats/charts.cjsx

```diff
@@ -124,7 +124,17 @@ Graph = React.createClass
         if data.index % numberBars
           data.element.attr({style: "display: none"})
     else if data.type == 'bar'
-      data.element.attr({style: "stroke-width: #{100 / length}%"})
+      data.element.attr(
+        style: "stroke-width: #{100 / length}%"
+        focusable: true
```

Comment:
No harm in having both. If a parser encounters an unknown attribute, it should just ignore it and move on (it may add the attribute as a property to the DOM node.)